From 155768e794e31892037e1a082d8457f7f122b9f0 Mon Sep 17 00:00:00 2001
From: Laurent Carlier <lordheavym@gmail.com>
Date: Fri, 18 Mar 2016 15:10:37 +0100
Subject: [PATCH xserver] Revert "present: Requeue if flip driver hook fails
 and target MSC not reached"

This reverts commit eb5108b87017128f394ae31b5b7cd85dd8819bca.
---
 present/present.c | 14 --------------
 1 file changed, 14 deletions(-)

diff --git a/present/present.c b/present/present.c
index 55f6aa7..d6df54f 100644
--- a/present/present.c
+++ b/present/present.c
@@ -724,20 +724,6 @@ present_execute(present_vblank_ptr vblank, uint64_t ust, uint64_t crtc_msc)
             if (window == screen_priv->flip_window)
                 present_unflip(screen);
         }
-
-        /* If present_flip failed, we may have to requeue for the target MSC */
-        if (msc_is_after(vblank->target_msc, crtc_msc) &&
-            Success == present_queue_vblank(screen,
-                                            vblank->crtc,
-                                            vblank->event_id,
-                                            vblank->target_msc)) {
-            xorg_list_add(&vblank->event_queue, &present_exec_queue);
-            xorg_list_append(&vblank->window_list,
-                             &present_get_window_priv(window, TRUE)->vblank);
-            vblank->queued = TRUE;
-            return;
-        }
-
         present_copy_region(&window->drawable, vblank->pixmap, vblank->update, vblank->x_off, vblank->y_off);
 
         /* present_copy_region sticks the region into a scratch GC,
-- 
2.7.3

